style.css

```diff
@@ -1,9 +1,331 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #1C1D21;
+}
+/* TODO: never set exact width on body */
 
-.header{
-    display:flex;
+.header {
+    display: flex;
+    height: 30px;
+}
+
+.nav {
+    display: flex;
     flex-direction: row;
```

Comment:
row is default setting, you can delete it. If you set display to flex, the flex-direction value is automatically set as `row`